### ISSUE TITLE
Allow reordering of the favorites icons

### ISSUE THREAD

**Issue by Queuecumber** [zpydr/gnome-shell-extension-taskbar#43]
Allow favorites icons to be reordered by clicking and dragging (enhancement request)

**Comment by rgeary1** [comment]
+1

**Comment by feraudet** [comment]
+1

**Comment by sebalis** [comment]
Yes, please! This is a function I seriously miss. I keep thinking about the order I start applications to make them appear in the desired order in the task bar. Sometimes I find myself closing “old” applications and restart them just in order to get them to right position in the task bar (or after the order is changed after a restart, see issue #114). I know this is just a personal obsession but I wonder if I’m alone with it. Being able to drag and drop would certainly save me time and free some mental capacity.

**Comment by rogerweb** [comment]
> I wonder if I’m alone with it

You are not. I've been searching for a task bar with this capability (reorder ungrouped windows) for days now with no success.
